src/index.html

```diff
@@ -5,9 +5,116 @@
+            <p>The procedural change mezzo forte starts izoritmichesky pickup at these moments stop LA Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+          </div>
+          <form action="" class="form consultation__form">
+            <label><input type="text" name="name" placeholder="Name" class="input form__input"></label>
+            <label><input type="text" name="phone" placeholder="Phone" class="input form__input"></label>
+            <label><input type="text" name="email" placeholder="Email" class="input form__input"></label>
+            <button class="button form__button">Consultation</button>
+          </form>
+        </div>
+      </section>
+      <section class="section">
+        <h2 class="heading section__heading">Testimonialis<span class="heading__crown"></span></h2>
+        <div class="testimonials section__content">
+          <div class="person">
+            <img class="person__photo" src="images/people/1.jpg" alt="Tim Cook">
+            <h3 class="person__name">Tim Cook</h3>
+            <p class="person__position">CEO</p>
+            <p class="person__text">Nice work, Certificates National Association of Realtors (USA).</p>
+          </div>
+          <div class="person">
+            <img class="person__photo" src="images/people/2.jpg" alt="Katherine Adams">
+            <h3 class="person__name">Katherine Adams</h3>
+            <p class="person__position">Senior Vice President</p>
+            <p class="person__text">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart. I’m all alone and blissfully happy in the local edge.</p>
+          </div>
+          <div class="person">
+            <img class="person__photo" src="images/people/3.jpg" alt="Jonathan Ive">
+            <h3 class="person__name">Jonathan Ive</h3>
+            <p class="person__position">Chief Design Officer</p>
+            <p class="person__text">When my lovely valley of steam rises and half-day sun is above an impermeable.</p>
+          </div>
+        </div>
+      </section>
+      <section class="section">
+        <h2 class="heading section__heading">Contact Us<span class="heading__crown"></span></h2>
+        <div class="contact">
+          <div class="contact_text">
+            <p><a href="mailto:hello@miami.com" class="link">Hello@miami.com</a><br>+1 234 555-55-55</p>
+            <p>400 first ave. n.<br>suite 700<br>Minneapolis, MN 55401</p>
+          </div>
+          <form class="form contact__form">
+            <label><input type="text" name="name" placeholder="Name" class="input form__input"></label>
+            <label><input type="text" name="email" placeholder="Email" class="input form__input"></label>
+            <label><textarea name="message" placeholder="Message" class="input input__textarea form__input"></textarea></label>
```

Comment:
переноси длинные строки
```suggestion
            <label>
              <textarea 
                name="message" 
                placeholder="Message" 
                class="input input__textarea form__input"
              ></textarea>
            </label>
```